aws/ecs/variables.tf

```
@@ -0,0 +1,22 @@
+variable "license_key" {
+  type      = string
+  sensitive = true
+  nullable  = false
+}
+
+variable "docker_token" {
+  type      = string
+  sensitive = true
+  nullable  = false
+}
+
+variable "environments_management_secret_key" {
+  type      = string
+  sensitive = true
+  nullable  = false
+}
+
+variable "image_version" {
+  type     = string
+  nullable = false
+}
```

What about default version set to latest? :D 
```suggestion
variable "image_version" {
  type     = string
  default  = "latest"
}
```

---

Using the default value of `latest` here has some dangers of using that version by the script users even though we do not recommend it. Instead of making the version change as an option/one of the steps we are now requiring the script user to set the value by themself. If someone would use `latest` here, that is their decision and not a side-effect of script usage

---

We are already defaulting to `latest` tag in other configurations with a mention in the documentation that it isn't recommended. Like here: https://github.com/cksource/ckeditor-cs-on-premises-infrastructure/tree/main/kubernetes/helm/ckeditor-cs#installation